src/features/session_replay/aggregate/index.js

```diff
@@ -301,7 +303,7 @@ export class Aggregate extends AggregateBase {
   }
 
   getCorrectedTimestamp (node) {
-    if (!node.timestamp) return
+    if (!node?.timestamp) return
```

Comment:
So if there is a node without a timestamp?? I thought that wasn't possible.

Reply:
The only time i saw this come up was in a jest test, but i just added it while I was thinking about it